### DISCUSSION TITLE
Box plot in Seq

### DISCUSSION THREAD

**Discussion by emanuelefirmani** [datalust/seq-tickets#2267] [Help]
I'd like to have a dashboard showing a [box plot](https://en.wikipedia.org/wiki/Box_plot) of the execution times of my application, something like:
![box_plot_thumbnail](https://github.com/user-attachments/assets/5f0ea462-ade4-4af5-baa3-b7b2b346bbe0)

If not possible, I would go for a close solution using stacked bars:
![image](https://github.com/user-attachments/assets/e34bdddb-9aeb-4178-bb4a-16601d6dec32)

How can I achieve the expected result?
Thanks for your help

**Comment by nblumhardt** [comment]
Hi! There isn't much control over formatting for this in Seq, but you can `select percentile(@Elapsed, 90), percentile(@Elapsed, 75), percentile(@Elapsed, 50)` as three columns and use a line plot to show the three bands with this kind of rendering:

![image](https://github.com/user-attachments/assets/5853e98e-3fd1-4659-9d98-a99485fefe1b)

Hope this helps,
Nick

**Reply by emanuelefirmani** [reply]
I tried using a dotted plot, but the result wasn't perfect. I'll give your proposal a shot, although it works well only if you have a single series. 
Thanks!
